extensions/python/pythonprocessors/nifiapi/flowfiletransform.py

```diff
@@ -98,12 +98,15 @@ def onTrigger(self, context: ProcessContext, session: ProcessSession):
             session.transfer(original_flow_file, self.REL_FAILURE)
             return
 
+        result_attributes = result.getAttributes()
         if result.getRelationship() == "failure":
             session.remove(flow_file)
+            if result_attributes is not None:
+                for attribute in result_attributes:
+                    original_flow_file.addAttribute(attribute, result_attributes[attribute])
```

Comment:
I think
```suggestion
                for name, value in result_attributes.items():
                    original_flow_file.addAttribute(name, value)
```
would be nicer

Reply:
Updated in 30268333d5fff6c48822cb54925b8189ff6d0e49